docker-compose.e2e.yaml

```diff
@@ -0,0 +1,52 @@
+version: "3.9"
+services:
+  synpress:
+    container_name: synpress
+    build:
+      dockerfile: ./Dockerfile.e2e
+      context: .
+    depends_on:
+      - display
+      - video
+    entrypoint: []
+    working_dir: /app
+    volumes:
+      - ./docker/videos:/app/public-report/screenshots
+      - ./docker/screenshots:/app/public-report/videos
+    environment:
+      - DISPLAY=display:0.0
+      - SYNDEBUG=true
+      - CYPRESS_DOCKER_RUN=true
+      - TURBO_TOKEN=${TURBO_TOKEN}
+      - TURBO_TEAM=${TURBO_TEAM}
+    command: >
+      bash -c 'yarn wait-on http://display:8080 && yarn test:e2e'
+    networks:
+      - x11
+
+  display:
+    container_name: display
+    image: synthetixio/display:b2643097e891906524e52e7ee956260b20fa01fb-base
+    environment:
+      - RUN_XTERM=no
+      - DISPLAY_WIDTH=1920
+      - DISPLAY_HEIGHT=1080
+    ports:
+      - "8080:8080"
+    networks:
+      - x11
+
+  video:
```

Comment:
Is this needed in the CI?

Reply:
I should have removed this in 153436a53cfa67f5ca3970a5b251c03538e30a94